src/Workspaces/Core/Portable/Utilities/NullableHelpers/AllNullabilityIgnoringSymbolComparer.cs

```diff
@@ -8,7 +8,6 @@ namespace Microsoft.CodeAnalysis
     /// <summary>
     /// A comparer that compares symbols but ignores nullability, both top-level and nested.
     /// </summary>
-    /// <remarks>This is not implemented correctly, which is being tracked by https://github.com/dotnet/roslyn/issues/36044</remarks>
     internal sealed class AllNullabilityIgnoringSymbolComparer : IEqualityComparer<ISymbol>
```

Comment:
dumb question.  why do we need this?  i thought the compiler already had a comparer or .Equals method for this sort of comparison...

Reply:
This also will ignore the top-level nullability in our wrapper. This will indeed go away once the compiler moves to that model.

Reply:
Gotcha.  Thanks!